benchmarks/graphics_pipeline/GraphicsBenchmarkApp.cpp

```diff
@@ -53,6 +53,22 @@ void GraphicsBenchmarkApp::InitKnobs()
     pDrawCallCount->SetDisplayName("DrawCall Count");
     pDrawCallCount->SetFlagDescription("Select the number of draw calls to be used to draw the `sphere-count` spheres.");
 
+    pAlphaBlend = GetKnobManager().CreateKnob<ppx::KnobCheckbox>("alpha-blend", false);
```

Comment:
Can you also change the `Indent` of all the knobs related to the spheres to 1? Similar to how the quads related knobs are indented and displayed below the `pFullscreenQuadsCount` knob.

Reply:
Done.